letsdrill.rb

```diff
@@ -1,13 +1,21 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  if integer >= 90
+  	return "A"
+  elsif integer >= 80 and integer <= 89 
+  	return "B"
+  elsif integer >= 70 and integer <= 79 
+  	return "C"
+  elsif integer >= 60 and integer <= 69 
+  	return "D"
+  elsif integer >= 0 and integer <= 59 
+  	return "E"
```

Comment:
Turns out, you don't really need the `return`s up here either. The whole `if`/`elsif` evaluates to whichever branch is run.

This would be equivalent:
```ruby
return if integer >= 90
         "A"
       elsif ...
```
which is the same as no `return`.